blink/tui.c

```diff
@@ -1065,20 +1072,24 @@ static void DrawTerminalHr(struct Panel *p) {
     }
     AppendStr(&p->lines[0], "\033[1m");
   }
+  wp = p->right - p->left;
+  ws = 8;
+  wl = wp / 4 - ws / 2;
+  for (i = 0; i < wl; ++i) AppendWide(&p->lines[0], HR);
 #ifdef IUTF8
```

Comment:
If you like, I can remove the entire \#ifdef for you.

I had thought the only reason for IUTF8 in the kernel was for it to know how to handle *incoming* backspaces properly, so that they erase properly. With regards to other output (except for certain shells or within enhanced readline's etc) I'm not under the understanding that the kernel tracks the cursor location (but of course terminal emulators do).

I saw some of the special \e[y;xH emits in the pty driver and I think elsewhere, so without these you're seeing bad output, only on BSD kernels? Does BSD have IUTF8 defined? Other possible confusion is the difference between OS's terminal emulator on BSD and Linux/macOS, ?

Reply:
Suspected IUTF8 problem fixed in last push.